app/javascript/run_export.js

```diff
@@ -0,0 +1,28 @@
+const setRoutesOnly = function(routesOnly) {
+  Array.from(document.getElementById('export-menu').getElementsByClassName('can-strip-history')).forEach((el) => {
+    if (routesOnly) {
+      el.href = el.href + '?blank=1'
+    } else {
+      el.href = el.href.replace('?blank=1', '')
+    }
+  })
+}
+
+document.addEventListener('turbolinks:load', () => {
+  checkbox = document.getElementById('route-check')
+  if (checkbox === null) {
+    return
+  }
+
+  setRoutesOnly(checkbox.checked)
+
+  checkbox.addEventListener('change', (event) => {
+    setRoutesOnly(event.target.checked)
+  })
+
+  // Stop clicks on the checkbox's label text from closing the dropdown menu
+  document.getElementById('route-check-label').addEventListener('click', (event) => {
```

Comment:
This one actually doesn't look like we can, but also looks like we don't need to.

We *can't* because making a listener like
```javascript
document.addEventListener('click', event => {
  if (event.target.matches('#route-check-label')) {
    event.stopPropagation()
  }
})
```
doesn't stop the propagation in time, because the event bubbles up to `document` _after_ Bootstrap has responded to it by closing the dropdown. So the net effect is the dropdown still closes when clicking on the "Route only" label.

We *don't need to* because attaching two or three or four listeners whose content is just
```javascript
function(event) {
  event.stopPropagation()
}
```
doesn't hurt anything in the end, as even if a user navigates back and forth like a hundred thousand times without a hard load, the function would still only be called once and the `stopPropagation()` would cancel the others. It's a bit messy for sure, but I don't think it will ever have any real measurable effect.

The alternative is to fix this label-closes-the-dropdown issue by completely disabling Bootstrap's autoclose behavior and rebuilding it from scratch with this one checkbox's label as the exception. And I'm confident I'd screw that up much worse than the small morale (maybe super tiny memory?) hit of the above solution.